### ISSUE TITLE
Remove support for deprecated concept names

### ISSUE THREAD

**Issue by Ndpnt** [MattiSG/Watai#143]
In the version 0.7 there are the following renaming:
- Hook → Locator
- Widget → Component
- Data → Fixture
- Feature → Scenario
- Flow → Verbose

Support for previous concepts names have to be removed in the major release v1.0

**Comment by Ndpnt** [comment]
@MattiSG Are you ok with this?

**Comment by MattiSG** [comment]
Yes, that’s great, thanks a lot for marking it, I think it's a great way to do it!  :))
